README.md

```diff
@@ -139,6 +139,60 @@ To run the tests, execute the following command from the repo's root directory:
 ./scripts/run.e2e.sh
 ```
 
+## Snapshots usage for local networks
+
+Network snapshots are used by the CLI in order to keep track of blockchain state, and to improve performance of local operations.
```

Comment:
I think `local operations` is not very clear. It could be misunderstood as "local" as your machine (I understood it first like that).

I suggest to try making it clear that it's about local _deployments_